samples/caf/configuration/nrf52840dk_nrf52840/buttons_def.h

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Maybe we could place buttons_def and led_state def in a common directory to avoid code duplication ? (in our other applications we also use common configuration directory)

Reply:
I had the configuration for Thingy:53 as well and if we decide to add the support for this board too the `led_state_def.h` files would be different. I know right now its the same, but adding any board that has button connected to gpio from port 1 would lead to splitting this as is right now.

Reply:
I think keeping this separate is cleaner for the user.

Reply:
In case we would like to add support from boards other than DKs in the future then it may be simpler to leave it as it is (at least for now)